### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**

### CODE REVIEW

**Review by generatedman — APPROVED** [review]
good job!

**Review by naziamoff — CHANGES_REQUESTED** [review]
let's avoid using things like `values[0]` here. It makes your code difficult to read as I don't understand what exactly is stored there

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,28 @@
...
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const data of action.keysToRemove) {
```
Comment (naziamoff) [inline src/transformState.js]: is this `data` or a `key`?

**Review by naziamoff — APPROVED** [review]
looks good to me 👍

**Pull request closed without merging**